src/it/scala/djinni/IntegrationTest.scala

```diff
@@ -46,8 +55,12 @@ class IntegrationTest extends FunSpec {
     djinni("--java-package djinni.it " +
       s"--java-out src/it/resources/result/$idl/java " +
       s"--cpp-out src/it/resources/result/$idl/cpp " +
+      s"--cpp-header-out src/it/resources/result/$idl/cpp-headers " +
       s"--jni-out src/it/resources/result/$idl/jni " +
+      s"--jni-header-out src/it/resources/result/$idl/jni-headers " +
       s"--objc-out src/it/resources/result/$idl/objc " +
+      s"--objc-header-out src/it/resources/result/$idl/objc-headers " +
+      "--objc-swift-bridging-header bridging-header " +
```

Comment:
I added the `header-out` parameters to the output-file test to ensure that all parameters work. Also I added the `bridging-header` parameter so the test can check for the correct output directory for the header file.